.github/workflows/main.yml

```
@@ -282,28 +282,32 @@ jobs:
         with:
           name: ubuntu-latest-3.12
           path: /tmp/u312
+      - uses: actions/download-artifact@v4
+        with:
+          name: ubuntu-latest-3.13
+          path: /tmp/u313
       - uses: actions/download-artifact@v4
         with:
           name: macos-latest-3.9
           path: /tmp/m39
       - uses: actions/download-artifact@v4
         with:
-          name: macos-latest-3.12
-          path: /tmp/m312
+          name: macos-latest-3.13
+          path: /tmp/m313
       - uses: actions/download-artifact@v4
         with:
           name: windows-latest-3.9
           path: /tmp/w39
       - uses: actions/download-artifact@v4
         with:
-          name: windows-latest-3.12
-          path: /tmp/w312
+          name: windows-latest-3.13
+          path: /tmp/w313
       - name: Install Dependencies
         run: pip install -U coverage coveralls diff-cover
         shell: bash
       - name: Combined Deprecation Messages
         run: |
-          sort -f -u /tmp/u39/ml.dep /tmp/u310/ml.dep /tmp/u311/ml.dep /tmp/u312/ml.dep /tmp/m39/ml.dep /tmp/m312/ml.dep /tmp/w39/ml.dep /tmp/w312/ml.dep || true
+          sort -f -u /tmp/u39/ml.dep /tmp/u310/ml.dep /tmp/u311/ml.dep /tmp/u312/ml.dep /tmp/m39/ml.dep /tmp/m313/ml.dep /tmp/w39/ml.dep /tmp/w313/ml.dep || true
```

```suggestion
          sort -f -u /tmp/u39/ml.dep /tmp/u310/ml.dep /tmp/u311/ml.dep /tmp/u312/ml.dep /tmp/u313/ml.dep /tmp/m39/ml.dep /tmp/m313/ml.dep /tmp/w39/ml.dep /tmp/w313/ml.dep || true
```
Is missing the newly added ubuntu 3.13